tax_determination.rb

```diff
@@ -0,0 +1,13 @@
+module TaxDetermination
+  def is_exempt?(product)
+    products = %w(pills book books chocolate chocolates)
+    bool = false
+    products.each do |i|
+      if product.include?i
+        bool = true
+        break
+      end
+    end
+    bool
```

Comment:
```suggestion
    return (product.split & products).present?
```
Will this return the correct value?